bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/contract/form/helper/FormUtil.java

```diff
@@ -0,0 +1,20 @@
+package org.bahmni.module.bahmnicore.contract.form.helper;
```

Comment:
This should go under "org.bahmni.module.bahmnicore.util" package. Also no need to create additional "helper" package.